docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -2053,6 +2053,27 @@ options:
 
 Both are equally fine, just a matter of preference.
 
+#### Waiting for Storage to load
```

Comment:
@CTNicholas Do you prefer to move this subsection down? If so, where do you prefer it?

Reply:
Need to add props tables for all the React hooks when I have a chance.